etc/views.sql

```diff
@@ -49,10 +49,11 @@ GROUP BY w.cldf_languageReference;
 CREATE VIEW IF NOT EXISTS wordstats AS                                 
 SELECT                                                             
 	w.cldf_languageReference,
+	w.cldf_name,
 	-- word form frequency
-	count(w.cldf_name) as WordFreq
+	count(w.cldf_id) as WordFreq
 FROM
 	'words.csv' as w
-GROUP BY w.cldf_languageReference, cldf_name;
+GROUP BY w.cldf_languageReference, w.cldf_name;
```

Comment:
@Tarotis I suppose that's what we want here, right? Number of words per (form, language)?

Reply:
ups, yes, I forgot the w. because I was trying out the code within the table itself